src/app/workspace/cmdinput/textareainput.tsx

```diff
@@ -604,17 +604,53 @@ class TextAreaInput extends React.Component<{ screen: Screen; onHeightChange: ()
                 }
             }
         }
-        const isHistoryFocused = auxViewFocused && inputModel.getActiveAuxView() == appconst.InputAuxView_History;
+
+        const renderCmdInputKeybindings = mobx
+            .computed(() => {
+                const inputModel = GlobalModel.inputModel;
```

Comment:
Same as above, we should make this a reusable function that can accept a aux view type and return whether the keybindings should be set.

Reply:
great idea, addressed